nva/nvafakebios.c

```diff
@@ -214,8 +218,9 @@ void edit_bios(uint8_t *vbios, size_t vbios_length, struct edit_offset *edit)
 
 int main(int argc, char **argv) {
 	uint8_t *vbios = NULL;
+	uint64_t vbios_vram = 0;
 	size_t vbios_length = 0;
-	int c, i, cnum = 0, result = 0;
+	int c, i, cnum = 0, result = 0, vram_size=0;
```

Comment:
declare vram_size as uint64_t to protect against overflows

Reply:
Is it needed when it's only user input?

Reply:
you assign the adjusted value to it.

Reply:
only used as user input, but thanks for the suggestion that (vram_size * 1024) result is still 32 bit